pkg/gui/gui.go

```diff
@@ -267,13 +267,34 @@ func (gui *Gui) Run() error {
 		gui.waitForIntro.Add(1)
 	}
 
+	updates := gui.Config.UserConfig.Update
+	projectDuration := time.Millisecond * 100
```

Comment:
I would set the defaults in app_config rather than here. We're currently doing it for max duration of stats collection, but we could clean that up by doing:
```
func mustParseDuration(durationStr string) time.Duration {
	duration, err := time.ParseDuration(durationStr)
	if err != nil {
		panic(err)
	}
	return duration
}
```

and then doing something like:
```
		Update: UpdateConfig{
			RefreshRate: mustParseDuration("100ms"),
		},
```

Reply:
This is indeed cleaner, i'll change it to that.